### PR TITLE
Create forms for location search

### PR DESCRIPTION
**Pull request by Polyanyanwu**
# Business Analysis 🟠 Code Institute 🟠 Community Project - Pull Request 🎉🎉

---

## Pull Request References:
- Created form to receive user selection of location 

## PR contains:
- Updated view_home from home.py to receive the user location criteria and send back to the UI
- Updated home.html to include forms to accept user location as search criteria

## Breaking changes
- None
- However, there is need for collaboration with Evelyn to harmonize the look of the various search criteria on the home page.

## Screenshots/Gifs/Media
image

### CODE REVIEW

File: app/templates/home/home.html [hunk 1]
```diff
@@ -1,45 +1,124 @@
...
-
-{% block heading %}
-    <h1>Home page</h1>
-{% endblock %}
-
-{% block content %}
-<button id="filterPanel" class="btn btn-primary" type="button" data-bs-toggle="collapse" data-bs-target="#collapseFilter"
-    aria-expanded="false" aria-controls="collapseFilter">
-    Open Filter Panel
+{% extends "base.html" %} {% block heading %}
```
Comment (stefdworschak) [inline app/templates/home/home.html:1]: Why did you collapse all the tags below? It looks a bit squished together now. Could you please put this back as it was.

**Review by stefdworschak — CHANGES_REQUESTED** [review]
@Polyanyanwu is this a change to the UI only? I can't really see this filtering any results.

File: app/templates/home/home.html (hunk 1 above)
Comment (Polyanyanwu) [reply app/templates/home/home.html:1]: We need to agree how those search fields will be arranged. I simply put my own below the existing ones, I did comment that there is need to harmonize it. That can happen even after the functionality is fully tested

**Comment by auxfuse** [comment]
@stefdworschak @Polyanyanwu as this PR is a good bit out of date now, shall we close this off without merging? The project has moved on quite a bit since this has been submitted and I'm just wondering if we should "clear the slate" so to speak.

**Comment by Polyanyanwu** [comment]
Please go ahead. I'm so sorry I couldn't continue with the project.

**Comment by auxfuse** [comment]
Thanks for the update @Polyanyanwu 😄 Please don't apologise though. We were delighted to have your input into this project in any capacity at all. I've closed this without merging. 🤜

**Pull request closed without merging**